tests/cnf/ran/internal/rancluster/rancluster.go

```diff
@@ -125,3 +126,17 @@ func WaitForNumberOfNodes(client *clients.Settings, expected int, timeout time.D
 			return false, nil
 		})
 }
+
+// CheckSpokeClusterType checks and returns a spoke cluster type based on number of control plane nodes.
+func CheckSpokeClusterType(client *clients.Settings) (string, error) {
+	controlPlaneNodesList, err := ListNodesByLabel(client, RANConfig.ControlPlaneLabelMap)
+	if err != nil {
+		return "", err
+	}
+
+	if len(controlPlaneNodesList) == 1 {
+		return ranparam.SNO, nil
+	}
+
+	return ranparam.Standard, fmt.Errorf("could not determine spoke cluster type")
```

Comment:
for the error case, could you make it return `""` instead of standard?

Reply:
Okay, updated. Thanks!